easybuild/easyconfigs/x/xalt/xalt-2.7.29.eb

```diff
@@ -0,0 +1,61 @@
+easyblock = 'EB_XALT'
+
+name = 'xalt'
```

Comment:
@samcmill We should use `XALT`, which is the official software name?

When doing so, the `easyblock = ` line can be removed, since EasyBuild will correctly derive the easyblock to use from the software name.